packages/core/admin/admin/src/pages/MarketplacePage/tests/index.test.js

```diff
@@ -262,9 +262,28 @@ describe('Marketplace page', () => {
     expect(notInstalledText).toBeVisible();
   });
 
-  it('Its showing the sort by menu and its options', async () => {
+  it('Its showing the sort by menu', async () => {
```

Comment:
To be more consistent with other tests, try to integrate the 'it' into your test name:
```suggestion
  it('shows the sort by menu', async () => {
```